src/agent/utils.ts

```diff
@@ -29,6 +29,7 @@ async function checkAgentRunning(nodePath: string): Promise<boolean> {
 async function spawnBackgroundAgent( // FIXME, this is broken.
```

Comment:
In this branch the tests for spawning an agent are passing so I don't know if this is still relevant

Reply:
I've removed this